### ISSUE TITLE
Enemy pathing non-functioning on destructible block objects.

### ISSUE THREAD

**Issue by Phanterm** [frogatto/frogatto#181]
![pathing](https://cloud.githubusercontent.com/assets/3878045/9700573/2eedb6a2-53cd-11e5-8207-492cf13e5485.gif)

**Comment by rkettering** [comment]
I suspect what we're seeing here is that because both involved parties are hittable objects, at any point where they contact such a surface with any y-velocity, even if it's just a single pixel's worth of displacement (or just descending a slope onto them), they trigger an elastic collision.

Over, and over.

If that's indeed the issue, it should be fairly straightforward to disable.

**Comment by rkettering** [comment]
This behavior, for anything other than objects that we really, emphatically *want* to be standable, is actually a really big boon - it's a nice, clean mechanism for resolving cases where two objects accidentally get situated on top of each other (due to whatever gameplay shenanigans, whether it's tricking an enemy into walking off a ledge, or some indestructible plant accidentally landing on top of another indestructible plant.)

Basically the best way to resolve this will be some sort of flag for a hittable object where we very directly indicate that it's meant to be standable, and in all other cases, we do the bounce seen above.
